src/tools/dart2native.md

```diff
@@ -1,7 +1,105 @@
 ---
 title: dart2native
 description: Command-line tool for AOT-compiling Dart source code.
-toc: false
 ---
 
-Coming soon.
+Use the `dart2native` command to AOT (ahead-of-time) compile
+a Dart program to native x64 machine code.
+The output of `dart2native` is either
+a **standalone executable** (the default)
+or an **AOT snapshot** that you can run with the [`dartaotruntime` command][].
+The `dart2native` command is supported on Windows, macOS, and Linux.
```

Comment:
Is it worth explaining what situations that an AOT snapshot would be useful over a standalone executable? Currently, with what little I know, it feels like AOT snapshot is the equivalent of a `.o` file. Is the AOT snapshot version independent?

Reply:
We could, for example something like "AOT snapshots can be helpful when you are limited on disk space, as they allow for running multiple AOT snapshots without each incurring the full AOT runtime disk overhead."

Reply:
I had this down below, but it's too easy to miss. I've removed that section and put all that content in the intro.